src/compiler/types/update-import-refs.ts

```diff
@@ -81,8 +89,23 @@ const updateImportReferenceFactory = (typeCounts: Map<string, number>, filePath:
           // If local then import location is the current file
         } else if (typeReference.location === 'local') {
           importResolvedFile = filePath;
-        } else if (typeReference.location === 'import') {
-          importResolvedFile = typeReference.path;
+        } else {
```

Comment:
I _believe_ it's okay to remove the `else if` block here since the only valid values for `location` are `local`, `global`, and `import`. But lemme know if anyone thinks this is risky: only did it to remove some SNC violations

Reply:
Yah - I think this is OK as well.